app/controllers/v0/preferences_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module V0
+  class PreferencesController < ApplicationController
+    before_action :set_preference
+
+    def show
+      render json: @preference
+    end
+
+    private
+
+    def set_preference
+      @preference = Preference.find_by(code: preference_params['code'])
+      raise Common::Exceptions::RecordNotFound, preference_params['code'] if @preference.blank?
```

Comment:
What are your thoughts about abstracting out the `preference_params['code']` calls on lines 14 and and 15 into a private `code` method, i.e.

```ruby
def code
  preference_params['code']
end
```

When I swap out line 14 for the new Redis call in my PR, it gets longer, and just looks nice and tidy with the `code` method, IMO.

Thoughts?